app/controllers/spree/admin/reports_controller_decorator.rb

```diff
@@ -244,7 +244,9 @@ def users_and_enterprises
   def xero_invoices
     if request.get?
       params[:q] ||= {}
-      params[:q][:completed_at_gt] = Time.zone.now.beginning_of_month
+      params[:q][:completed_at_gt] = Time.zone.today.beginning_of_month
+      params[:invoice_date] = Time.zone.today
+      params[:due_date] = Time.zone.today + 1.month
```

Comment:
Looking good! Small comment here (even though I'm not that familiar with the reports): it looks like `invoice_date` and `due_date` get defaults as well in `xero_invoices_report.rb`, so this would be superfluous. [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself) it up!